app/connectors/CustomsFinancialsApiConnector.scala

```diff
@@ -86,19 +93,20 @@ class CustomsFinancialsApiConnector @Inject()(httpClient: HttpClient,
     }
   }
 
-  def retrieveRequestedGuaranteeTransactionsDetail(
-                                                    gan: String,
-                                                    onlyOpenItems: Boolean,
-                                                    from: LocalDate,
-                                                    to: LocalDate)(
-                                                    implicit hc: HeaderCarrier): Future[Either[GuaranteeResponses, Seq[GuaranteeTransaction]]] = {
+  def retrieveRequestedGuaranteeTransactionsDetail(gan: String,
+                                                   onlyOpenItems: Boolean,
+                                                   from: LocalDate,
+                                                   to: LocalDate)
+                                                  (implicit hc: HeaderCarrier): Future[Either[GuaranteeResponses, Seq[GuaranteeTransaction]]] = {
```

Comment:
This line looks like it's over the length limit to me

Reply:
Addressed in this [commit](https://github.com/hmrc/customs-guarantee-account-frontend/pull/86/commits/c169bbfb4c8efe54a4e6ea4b455973a6624ba249)